gradle.properties

```diff
@@ -4,8 +4,6 @@ version=swan-lake-preview4-SNAPSHOT
 ballerinaVersion=2.0.0-Preview3
 shortVersion=slp3
 stdlibSocketVersion=0.6.7
-stdlibEncodingVersion=1.0.1
-stdlibJwtVersion=1.0.1
```

Comment:
Lets leave this as it is. Since we have to differentiate between BC released ones and not.